### DISCUSSION TITLE
avrdude issues labeled as bug

### DISCUSSION THREAD

**Discussion by mcuee** [avrdudes/avrdude#1409] [General]
I will use this discussion to better track the openning issues labeld as `bug`.
https://github.com/avrdudes/avrdude/issues?q=is%3Aopen+is%3Aissue+label%3Abug

**Comment by mcuee** [comment]
As of today 21-June-2023, it is a good time for avrdude, as we have only four opening issues labeled as bug.

* #1221 
This is a low priority issue and I believe I have identified the root cause.

* #968 
This is not an issue with avrdude per se as it is the issue of the avrdude-libusb library that @mariusgreuel developed for avrdude MSVC Windows build. But since MSVC build is the official Windows build for avrdude, I list it here as an avrdude bug.

* #379 
<del> This will be closed once we merge PR #1406 since I believe it will fix this issue</del>

     Edit: this requires more work than PR #1406.

* #336 
This will be closed once we merge PR #1411.

Updated on 27-06-2023: one more issue added
* #1414
To me it is a bug but mayber there is an argument that it is rather an enhancement.

**Comment by stefanrueger** [comment]
@mcuee Great analysis. Had a closer look at #379: this will only partially solved by #1406. #336 will be gone soon once a recent PR #1411 is merged. That will conclude my contributions to bug fixes for v 7.2.

**Comment by mcuee** [comment]
I just created a new issue #1414.
* #1414

To me it is a bug but mayber there is an argument that it is rather an enhancement.

**Comment by mcuee** [comment]
Please also refer to the following discussion.
* https://github.com/avrdudes/avrdude/discussions/1444

**Comment by mcuee** [comment]
Now that the number of issues are around 25 and can be fit into single page. I will close this discussion.
